### PR TITLE
Update for moved resource files

### PR DESCRIPTION
**Pull request by zhut19**
nT related resource files are being removed from `strax_auxiliary_files` See XENONnT/strax_auxiliary_files#9 and XENONnT/private_nt_strax_aux_files#1

### CODE REVIEW

**Comment by ramirezdiego** [comment]
@zhut19 Notice the repo is now called `private_nt_aux_files`.

**Comment by zhut19** [comment]
Ah, right, thank you @ramirezdiego. But I should wait a bit for that pr merge, so this can pass the travis test.

File: wfsim/load_resource.py [hunk 1]
```diff
@@ -28,36 +28,41 @@ def __init__(self, config=None):
...
+                'noise_file': 'x1t_noise_170203_0850_00_small.npz',
             })
         elif config['detector'] == 'XENONnT':
             files.update({
                 'photon_area_distribution': 'XENONnT_spe_distributions.csv',
                 's1_pattern_map': 'XENONnT_s1_xyz_patterns_corrected_MCv3.1.0_disks.pkl',
                 's2_pattern_map': 'XENONnT_s2_xy_patterns_topbottom_corrected_MCv3.1.0_disks.pkl',
-                'photon_ap_cdfs': 'xnt_pmt_afterpulse_config.pkl.gz',
                 's2_luminescence': 'XENONnT_s2_garfield_luminescence_distribution_v0.pkl.gz',
+                'photon_ap_cdfs': 'xnt_pmt_afterpulse_config.pkl.gz',
```
Comment (jorana) [inline wfsim/load_resource.py:50]: This looks certainly useful to change

File: wfsim/load_resource.py [hunk 2]
```diff
@@ -28,36 +28,41 @@ def __init__(self, config=None):
...
 
         for k in set(config).intersection(files):
             files[k] = config[k] # Allowing user to replace default with specified files
+
         commit = 'master'   # Replace this by a commit hash if you feel solid and responsible
-        url_base = f'https://raw.githubusercontent.com/XENONnT/strax_auxiliary_files/{commit}/fax_files'
+        if config['detector'] == 'XENON1T':
+            url_base = f'https://raw.githubusercontent.com/XENONnT/strax_auxiliary_files/{commit}/sim_files'
+        elif config['detector'] == 'XENONnT':
+            url_base = f'https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/{commit}/sim_files'
```
Comment (jorana) [inline wfsim/load_resource.py:64]: This does not work as it is a private github repository. We should revert this and use the solution I propose in #69

**Review by jorana — CHANGES_REQUESTED** [review]

**Comment by zhut19** [comment]
Close as this is completely covered by #69

**Pull request closed without merging**